template.tpl

```diff
@@ -1305,6 +1365,60 @@ const setCookieFrom = (headers, data) => {
   );
 };
 
+/*
+ * Given name and value strings returns 'name=value'.
+ */
+const mkCookie = (name, value) => {
+  return makeString(name) + '=' + makeString(value);
+};
+
+/*
+ * Joins an array of cookie-pairs ('name=value' strings).
+ */
+const joinCookies = (args) => {
+  const separator = '; ';
+  return args.filter((c) => !!c).join(separator);
+};
+
+/*
+ * Creates a Cookie header value based on the cookiesToForward field
+ * of the tag configuration.
+ */
+const mkCookies = (tagConfig) => {
+  if (!tagConfig.forwardCookieHeaders) {
+    return '';
+  }
+
+  const cookiesToForward = tagConfig.cookiesToForward;
+  if (cookiesToForward && cookiesToForward.length > 0) {
+    const cookies = cookiesToForward
+      .map((row) => {
+        // cookieName cannot be empty (see validation rules)
+        const cookieName = row.cookieName;
+        const noDecode = row.decode === 'yes' ? false : true;
```

Comment:
nit: Why flip this boolean? `const decode = row.decode === 'yes' ? true : false;` is easier to read to me

Reply:
Done! I meant to match google's name (`noDecode`), but it indeed ended up unreadable.